.travis.yml

```diff
@@ -4,7 +4,7 @@ android:
  components:
     - platform-tools
     - tools
-    - build-tools-26.0.2
+    - build-tools-27.0.3
     - android-26
```

Comment:
Update this aswell.

Reply:
@m-murad Changing android to 27 is causing the travis to fail. Is there anything else that needs to updated/added?

Reply:
What is the error message?